src/mux-analytics.brs

```diff
@@ -1562,44 +1773,64 @@ function muxAnalytics() as Object
     "aggregate": "ag",
     "api": "ap",
     "application": "al",
-    "audio": "ao",
     "architecture": "ar",
     "asset": "as",
     "autoplay": "au",
+    "average": "av",
+    "bitrate": "bi",
     "break": "br",
+    "browser": "bw",
+    "bytes": "by",
+    "business": "bz",
+    "cached": "ca",
+    "cancel": "cb",
     "codec": "cc",
     "code": "cd",
     "category": "cg",
+    "changed": "ch",
+    "clicked": "ck",
+    "canceled": "cl",
     "config": "cn",
     "count": "co",
+    "counter": "ce",
     "complete": "cp",
-    "connection": "cx",
+    "creative": "cr",
     "content": "ct",
     "current": "cu",
-    "country": "cy",
```

Comment:
One more thing - since we're removing this, we should also remove line 1204 (https://github.com/muxinc/roku-mux/pull/62/files#diff-f8a28b3a86f5c1fc62377d8bda764fa1328924c652111eeb9a96d2a7300df053R1204), the one that says `props.player_country_code = deviceInfo.GetCountryCode()`. That value is completely ignored server-side, so we should stop sending it (otherwise we won't be able to minify `country` with the removal of this map line